ui/components/app/alerts/unconnected-account-alert/unconnected-account-alert.js

```diff
@@ -97,7 +97,7 @@ const UnconnectedAccountAlert = () => {
       footer={footer}
     >
       <ConnectedAccountsList
-        accountToConnect={selectedIdentity}
+        accountToConnect={accountToConnect}
```

Comment:
Why change this? I thought this alert was only shown if the current selected account was the same as the "account to connect".

Reply:
Ah, you are correct, I just realized that we could avoid finding the selected identity in this component since the `getAccountToConnectToActiveTab` selector returns `identities[selectedAddress]` ([ref](https://github.com/MetaMask/metamask-extension/blob/develop/ui/selectors/permissions.js/#L179-L201)). If `accountToConnect` is `undefined`, that's handled by the `ConnectedAccountsList` component [here](https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/app/connected-accounts-list/connected-accounts-list.component.js/#L74-L76).

Reply:
Can we revert this change then? Or is there some reason to keep it? I don't really follow why we'd want to avoid using the `getSelectedIdentity` selector here.

Reply:
This was resolved. We reverted the change.